packages/docs/docs.json

```diff
@@ -16955,6 +16959,10 @@
           "name": "search",
           "docs": "Content will be rendered in search area to left of buttons."
         },
+        {
+          "name": "short-app-title",
+          "docs": "Handle routing by nesting a route in the short app title."
```

Comment:
feel like we need to make clearer when to use this slot i.e. for use when you want a shorter title on smaller devices size. Also make it clear that you can use both. Not sure if this is the place though, or if it comes more under guidance on the site. Maybe we could add the new example to the site, with additional info on how\why it does what it does??

Reply:
Guidance site update is [here](https://github.com/mi6/ic-design-system/pull/694). It will need to be built locally to see it working, but hopefully it adds a bit more info